installvm.py

```diff
@@ -349,11 +349,20 @@ def createKickstart(self):
                 urlstring = "--url=nfs://"+vmParser.confparser('repo', 'RepoIP') + ':/var/www/html' + self.repoDir  
 
         if vmParser.args.fs_type == 'btrfs':
-            addksstring = "autopart --type=lvm --fstype=btrfs"
+            if vmParser.args.partition_type == 'plain':
+                addksstring = "autopart --type=plain --fstype=btrfs"
```

Comment:
can this be like 

addksstring = "autopart --type=%s--fstype=%s"  % (vmParser.args.partition_type,vmParser.args.fs_type)

we need not repeat this

Reply:
even we can have add a check tool and only serve  supported parameter like 

if vmParser.args.partition_type not in ['lvm', 'plain']:
   logging.info("Aborting Installation : as partition_type is not supported or not valid")
 
 if vmParser.args.fs_type not in ['xfs','ext4', 'btrfs']:
   logging.info("Aborting Installation : as file system type is not supported or not valid")